qutebrowser/config/configdata.py

```diff
@@ -766,6 +766,11 @@ def data(readonly=False):
              "required to exactly match the requested domain.\n\n"
              "Local domains are always exempt from hostblocking."),
 
+            ('enable-pdfjs', SettingValue(typ.Bool(), 'true'),
```

Comment:
Sure, this way was just more convenient when testing with `--temp-basedir` :wink: